package/_broom.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env 
```

Comment:
is this file a dup?

Reply:
yes was just to have a duplicate of previous version while i refactored.